### DISCUSSION TITLE
Mapping commands

### DISCUSSION THREAD

**Discussion by LaCaradeculito** [andmarti1424/sc-im#720] [Q&A] [answered]
Hi. I've been trying to map commands and values like `:cellcolor "bg=BLACK fg=CYAN"` with `:vmap "C"  ":cellcolor "bg=BLACK fg=CYAN"<cr>"` but it didn't work to me (syntax error), neither did any attempt to arrange quotation marks. Is my command wrong or is it there's no possible mapping for it?

Thanks very much.
Marcelo

**Comment by andmarti1424** [comment, marked as answer]
This shall work:
`vmap "w" ":cellcolor \"bg=BLACK fg=CYAN\"<cr>"`

**Reply by LaCaradeculito** [reply]
Ups!... the special characters :) Thank you, and thank you so much for this tool.
